README.md

```diff
@@ -422,6 +432,55 @@ When storing logs on disk, we recommend using Logrotate in truncate mode.
 
 You can use [jq](https://stedolan.github.io/jq/) to traverse JSON log files, when you're not running Kiev in *development mode*.
 
+### Suffixing `tree_path`
```

Comment:
I've rewritten this section, @stereobooster , would you have a look?

Reply:
(Note to self: it needs `##`, not `###`)

Reply:
I would say this text is too "enterprise". We have exactly one case to solve - why we need to describe it in this abstract terms. Jump straight to explanation: if you use Shoryuken, please use this method. We recommend to use "B", "D" etc for suffixes, to be the same as of our standard encoding for tree_path.